aptos-move/framework/aptos-framework/sources/zkid.move

```diff
@@ -92,9 +89,8 @@ module aptos_framework::zkid {
             max_zkid_signatures_per_txn,
             max_exp_horizon_secs,
             training_wheels_pubkey,
-            nonce_commitment_num_bytes,
             max_commited_epk_bytes,
-            max_iss_field_bytes,
+            max_iss_val_bytes: max_iss_field_bytes,
```

Comment:
nit: also rename the variable that provides the value?

Reply:
oh, yes.